src/lib/components/Swap/WrapErrorText.tsx

```diff
@@ -0,0 +1,29 @@
+import { Trans } from '@lingui/macro'
+import useNativeCurrency from 'lib/hooks/useNativeCurrency'
+
+export enum WrapInputError {
```

Comment:
WrapInputError should be defined with the hook that returns it, in useWrapCallback.tsx.
Also, could you just call it `WrapError`?

WrapErrorText should be defined separately (ie should stay in this file).
nit: Would also like WrapErrorText to be renamed to `WrapError`, but up to you.